vm/vm.go

```diff
@@ -171,11 +171,23 @@ func (vm *VM) ExecInstructions(sets []*bytecode.InstructionSet, fn string) {
 	cf.self = vm.mainObj
 	vm.mainThread.callFrameStack.push(cf)
 
+	// this is the final destination of a Goby error
```

Comment:
Suggestion:
> To Goby maintainers:
This is the final destination of Goby errors at VM level and we don't deal with the errors themselves at this point.
We only decide how the user program should react to them.

Reply:
```suggestion
      // here is the final destination of Goby errors at the VM level, and we don't deal with them at this point.
      // we only decide how the user program should react to them.
```